app/src/lib/git/reflog.ts

```diff
@@ -31,14 +34,26 @@ export async function getRecentBranches(
     return []
   }
 
-  const output = result.stdout
-  const lines = output.split('\n')
+  const lines = result.stdout.split('\n')
   const names = new Set<string>()
+  // exclude master from recent branches
+  const excludedNames = new Set<String>(['master'])
+
   for (const line of lines) {
     const result = regex.exec(line)
-    if (result && result.length === 2) {
-      const branchName = result[1]
-      names.add(branchName)
+    if (result && result.length === 4) {
+      const operationType = result[1]
```

Comment:
This won't be necessary if the regex above is changed. If there is a match, it is necessarily a branch rename.